### DISCUSSION TITLE
Is there a clearer and simpler explanation to “Variable Importance (VIMP) with Subsampling Inference”？

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#153] [Q&A]
Is there a clearer and simpler explanation to “Variable Importance (VIMP) with Subsampling Inference”？
I really want to figure out this algorithm，but there are so many Complex notation in the paper(shown below)，I am looking forward to a clearer and simpler explanation  like a diagram.
![image](https://user-images.githubusercontent.com/5112397/148320338-d57d332d-0af2-44d5-aa87-730b6f58c56d.png)

**Comment by ishwaran** [comment]
Yes, you can consult our paper:

https://web.ccs.miami.edu/~hishwaran/papers/IL.StatMed.2019.pdf
